src/patternfly/components/Masthead/masthead.scss

```diff
@@ -3,174 +3,108 @@ $pf-v5-c-masthead--breakpoint-map: build-breakpoint-map();
 $pf-v5-c-masthead--inset-map: build-spacer-map("none", "sm", "md", "lg", "xl", "2xl");
 
 :root {
-  --#{$masthead}--PaddingLeft: var(--#{$masthead}--inset);
-  --#{$masthead}--PaddingRight: var(--#{$masthead}--inset);
+  --#{$masthead}--inset: var(--pf-t--global--spacer--lg);
+  --#{$masthead}--PaddingBlockStart: var(--pf-t--global--spacer--lg);
+  --#{$masthead}--PaddingBlockEnd: var(--pf-t--global--spacer--md);
+  --#{$masthead}--PaddingInlineStart: var(--#{$masthead}--inset);
+  --#{$masthead}--PaddingInlineEnd: var(--#{$masthead}--inset);
+  --#{$masthead}--RowGap: var(--pf-t--global--spacer--sm);
   --#{$masthead}--BackgroundColor: var(--pf-t--global--background--color--secondary--default);
+  --#{$masthead}--BorderWidth: var(--pf-t--global--border--width--divider--default);
+  --#{$masthead}--BorderColor: var(--pf-t--global--border--color--default);
 
-  // Insets
-  --#{$masthead}--inset: var(--pf-t--global--spacer--md); // match page insets
-  --#{$masthead}--xl--inset: var(--pf-t--global--spacer--lg); // match page insets
+  // Masthead main
+  --#{$masthead}__main--BorderBottomWidth: var(--#{$masthead}--BorderWidth);
+  --#{$masthead}__main--BorderBottomColor: var(--#{$masthead}--BorderColor);
```

Comment:
Since we're updating some other vars to use inline/block, we could do the same here